src/main/resources/application.yaml

```
@@ -1,105 +1,123 @@
-
-test:
-  auth:
-    enabled: false
-    secret-key: ${TEST_AUTH_SECRET_KEY}
-    user-id: ${TEST_AUTH_USER_ID}
-
-app:
-  sync:
-    target-url: ${APP_SYNC_TARGET_URL:}
-    api-key: ${APP_SYNC_API_KEY:}
+server:
```

YAMLlint (1.38.0)

[error] wrong new line character: expected \n

(new-lines)